js_modules/dagit/packages/core/src/runs/RunTimeline.tsx

```diff
@@ -179,9 +193,13 @@ export const RunTimelineContainer = ({
       return {...accum, [job.key]: Math.min(...startTimes)};
     }, {} as {[jobKey: string]: number});
 
-    return jobs.sort((a, b) => earliest[a.key] - earliest[b.key]);
-  }, [data, start, end, jobsByKey]);
+    setJobsWithRuns(jobs.sort((a, b) => earliest[a.key] - earliest[b.key]));
+    setLoadedWindow(hourWindow);
+  }, [data, start, end, loading, jobsByKey]);
```

Comment:
I think this is probably complaining about `hourWindow` not being in the array here and it'd probably be safe to add.